x/distribution/client/cli/query.go

```diff
@@ -7,15 +7,38 @@ import (
 
 	"github.com/spf13/cobra"
 
+	"github.com/cosmos/cosmos-sdk/client"
 	"github.com/cosmos/cosmos-sdk/client/context"
+	"github.com/cosmos/cosmos-sdk/client/utils"
 	"github.com/cosmos/cosmos-sdk/codec"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/version"
-	distr "github.com/cosmos/cosmos-sdk/x/distribution"
 	"github.com/cosmos/cosmos-sdk/x/distribution/client/common"
 	"github.com/cosmos/cosmos-sdk/x/distribution/types"
 )
 
+// GetQueryCmd returns the cli query commands for this module
+func GetQueryCmd(storeKey string, cdc *codec.Codec) *cobra.Command {
+	distQueryCmd := &cobra.Command{
+		Use:                        types.ModuleName,
+		Short:                      "Querying commands for the distribution module",
+		DisableFlagParsing:         true,
+		SuggestionsMinimumDistance: 2,
+		RunE:                       utils.ValidateCmd,
+	}
+
+	distQueryCmd.AddCommand(client.GetCommands(
+		GetCmdQueryParams(storeKey, cdc),
```

Comment:
quoting Marko: 
>should this instead be QueryRoute ? if you see this function below, it takes the `queryRoute` as params. I believe they are usually the same as the `ModuleName` but we could rename this in order to not confuse devs

Reply:
yeah that make sense, I've modified all instances like this one to now saw `queryRoute`